cmd/retry.go

```diff
@@ -61,7 +61,7 @@ var globalRandomSource = rand.New(&lockedRandSource{
 // until the maximum retry attempts are reached. - this function is a fully
 // configurable version, meant for only advanced use cases. For the most part
 // one should use newRetryTimerSimple and newRetryTimer.
-func newRetryTimerWithJitter(unit time.Duration, cap time.Duration, jitter float64, doneCh chan struct{}) <-chan int {
+func newRetryTimerWithJitter(unit time.Duration, max time.Duration, jitter float64, doneCh chan struct{}) <-chan int {
```

Comment:
Do not change the variable name `cap`  - any good reason you change it?

Reply:
Because of https://golang.org/pkg/builtin/#cap same as https://golang.org/pkg/builtin/#len in another file.